Iterable/iterable-web-sdk
# Add Option to Handle Links Differently

## JIRA Ticket(s) if any

* [MOB-3820](https://iterable.atlassian.net/browse/MOB-3820)

## Description

Allows user to open links uniformly. This solves issue where Bee editor forces all links to open in new tab, but the customer not wanting that

## Test Steps

1. Create in-app with some links
    * one relative link (e.g `/about`)
    * one internal link (e.g. `http://localhost:8080/about`)
    * one external link (e.g. `https://google.com/about`)
2. Set SDK config option in `getMessages` call `{ handleLinks: 'open-all-new-tab' | 'open-all-same-tab' | 'external-new-tab' }`
3. Ensure the option you set does what it advertises
   * `open-all-new-tab` should open links in new tab
   * `open-all-same-tab` should open links in same tab
   * `external-new-tab` should open external links in new tab, internal links in same tab
